frontends/web/src/routes/buy/components/infobutton.tsx

```diff
@@ -0,0 +1,12 @@
+import { BuyInfo } from '../../../components/icon/icon';
+import style from './infobutton.module.css';
+
+const InfoButton = ({ onClick, ...props }: {onClick: () => void}) => {
+  return (
+    <button onClick={onClick} {...props} className={style.button}>
+      <BuyInfo />
+    </button>
+  );
+};
+
+export default InfoButton;
```

Comment:
Ok. Thought would make sense to keep it inside `buy` as we don't use it anywhere but here. But sure I can move it to `components` in case of future usage.